src/OnlineRetailPortal.Core/Models/Product.cs

```diff
@@ -23,7 +24,7 @@ public class Product
         public Nullable<DateTime> PurchasedDate { get; set; }
         public Address PickupAddress { get; set; }
 
-        IProductStore productStore;
+        IProductStore productStore = new MockProductStore();
```

Comment:
core models should be unaware of your implementations

Reply:
done